code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -161,16 +161,3 @@
 	if(M.a_intent in accepted_intents)
 		M.changeNext_move(click_speed_modifier)
 	.= FALSE
-
-/obj/item/clothing/gloves/fingerless/rapid/admin
```

Comment:
You sure you dont want to keep these as well?

Reply:
eh sure